id-repository/id-repository-identity-service/src/main/java/io/mosip/idrepository/identity/service/impl/IdRepoServiceImpl.java

```diff
@@ -718,5 +718,10 @@ private static String splitUinHash(String uinHash) {
 		String substringHash = StringUtils.substringAfter(uinHash, "_");
 		return StringUtils.isBlank(substringHash) ? uinHash : substringHash;
 	}
-	
+
+	@Override
+	public Uin updateIdentity(IdRequestDTO request, String uin, boolean overridedata) throws IdRepoAppException {
+		return updateIdentity(request, uin);
+	}
```

Comment:
As per the code, from controller which one is called first the IdRepoServiceImpl or IdRepoProxyServiceImpl ?

Reply:
From controller the call first go to IdRepoProxyServiceImpl and then to the IdRepoServiceImpl.

Reply:
Resolved